BlueShift-iOS-SDK/InApps/BlueShiftNotificationViewController.m

```diff
@@ -15,6 +15,7 @@
 
 @interface BlueShiftNotificationViewController () {
     BlueShiftNotificationCloseButton *_closeButton;
+    NSMutableDictionary *cachedImageData;
```

Comment:
are we storing the image data inside this variable and keeping it live throughout the app's lifecycle?
IF yes, when are we releasing this memory once the image is no longer used?

Reply:
this gets stored only for that in-app message. Once in-app gets dismissed, this cachedImageData variable will be released.